### ISSUE TITLE
Update to work with CORS?

### ISSUE THREAD

**Issue by drewbaker** [Quasimondo/QuasimondoJS#7]
Hey Quasimondo,

It would be awesome to update these blur scripts to work with CORS now:
http://blog.chromium.org/2011/07/using-cross-domain-images-in-webgl-and.html
http://html5-demos.appspot.com/static/html5-whats-new/template/index.html#14

Or maybe this: http://content-security-policy.com/

I've tried hacking away at it, but suck at Canvas. This didn't seem to work:

```
var img = document.getElementById( imageID );
img.crossOrigin = '';
var w = img.naturalWidth;
var h = img.naturalHeight;
```

**Comment by drewbaker** [comment]
I got it working like below, with the help of some guys from SO:
http://stackoverflow.com/questions/29358555/challenge-fix-a-canvas-plugin-to-work-with-cors/29360630

```
function stackBoxBlurImage( imageURL, $canvasObj, radius, blurAlphaChannel, iterations )
{

    // Cache vars
    var img = document.createElement('img');
    var canvas = $canvasObj[0];

    // Make it work with CORS
    img.crossOrigin = '';
    img.src = imageURL;     

    // Wait till image loads
    img.onload = function(e) {

        // Setup image vars         
        var w = img.naturalWidth;
        var h = img.naturalHeight;

        // Setup Canvas size
        canvas.style.width  = w + "px";
        canvas.style.height = h + "px";
        canvas.width = w;
        canvas.height = h;

        // Add image to Canvas
        var context = canvas.getContext("2d");  
        context.drawImage(img, 0, 0, canvas.width, canvas.height);
        context.clearRect( 0, 0, w, h );
        context.drawImage( img, 0, 0 );    

        // Abort if radius is invalid
        if ( isNaN(radius) || radius < 1 ) return;

        // Blur!
        stackBoxBlurCanvasRGB( canvas, 0, 0, w, h, radius, iterations );
    };
}
```

**Comment by Quasimondo** [comment]
Thanks for looking into this - I'll see if this can be integrated without breaking the original behaviour.
